Source/WebInspectorUI/UserInterface/Controllers/JavaScriptLogViewController.js

```diff
@@ -138,7 +138,7 @@ WI.JavaScriptLogViewController = class JavaScriptLogViewController extends WI.Ob
 
         function saveResultCallback(savedResultIndex)
         {
-            let commandResultMessage = new WI.ConsoleCommandResultMessage(result.target, result, false, savedResultIndex, shouldRevealConsole);
+            let commandResultMessage = new WI.ConsoleCommandResultMessage(result.target, result, false, savedResultIndex, !!shouldRevealConsole);
```

Comment:
i think it'd be better to have the `!!` be inside `WI.ConsoleCommandResultMessage` instead to catch all cases it could be provided instead of just this one